lib/mapping/CMapEditManager.h

```diff
@@ -58,8 +58,9 @@ template<typename T>
 class DLL_LINKAGE CMapSelection
 {
 public:
-	explicit CMapSelection(CMap * map) : map(map) { }
-	virtual ~CMapSelection() { };
+	explicit CMapSelection(CMap * map)
+		: map(map) {}
```

Comment:
Some imperfection here, but might be we need no constructors with initializers lists in headers?